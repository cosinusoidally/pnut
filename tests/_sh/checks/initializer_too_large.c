// expect_comp_failure
int arr[2] = {1, 2, 3};

void main() {
  return;
}
