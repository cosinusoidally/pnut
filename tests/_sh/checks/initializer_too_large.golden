tests/_sh/checks/initializer_too_large.c:3:0  Array type is too small for initializer
